app/models/issue.rb

```diff
@@ -12,6 +12,10 @@ def error
     ErrorStore.find(self)
   end
 
+  def data=(data)
+    super(ErrorStore::Utils.encode_and_compress(data))
+  end
+
```

Comment:
here you should overwrite the `def data` method too to decode the column info. So it's always decoded when we access it anywhere in the codebase. That way you don't have to decode it every time you wanna read it!